dali/operators/util/pad.cu

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <map>
+#include <vector>
+#include "dali/operators/util/pad.h"
+#include "dali/core/static_switch.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(Pad)
+    .DocStr(R"code(Pads all samples with `fill_value` in the given `axes`,
+to match the size of the biggest dimension on those axes in the batch.
+The element padding axes is specified with the argument `axes`.
+Supported types: int, float.
+Examples:
+- Batch of 3 1-d samples, fill_value=-1, axes=(0,)
```

Comment:
Nitpick - why the trailing comma?

Reply:
Why not? :stuck_out_tongue:
It's quite common to have trailing commas in Python and I followed @jantonguirao's [suggestion](https://github.com/NVIDIA/DALI/pull/1180#discussion_r336359950).